rawdata/misc/commanders_to_ach_generals.csv

```diff
@@ -0,0 +1,745 @@
+US80,http://dbpedia.org/resource/George_Henry_Thomas,6951057d-db32-4db5-8b3e-dea0c1101a5f
+CS22,http://dbpedia.org/resource/Daniel_Harvey_Hill,0b57b2da-99cd-48b9-8cbd-b843cd7ca04f
+CS69,http://dbpedia.org/resource/John_B._Magruder,256b790f-0cad-46d6-80aa-7e176890da83
+US50,NA,93f882f8-8924-4bfb-bbc2-4460c9841ea2
+CS111,http://dbpedia.org/resource/P._G._T._Beauregard,183d0a7d-f06a-4c7b-a575-7616e491618b
+US96,http://dbpedia.org/resource/Irvin_McDowell,0f05f7ed-f187-4469-8fbe-25942e7eb85b
+CS111,http://dbpedia.org/resource/P._G._T._Beauregard,183d0a7d-f06a-4c7b-a575-7616e491618b
+CS92,http://dbpedia.org/resource/Joseph_E._Johnston,b0f7a304-f6c5-4a8d-b816-066426e5dc1c
+US96,http://dbpedia.org/resource/Irvin_McDowell,0f05f7ed-f187-4469-8fbe-25942e7eb85b
+CS110,http://dbpedia.org/resource/Nathan_George_Evans,6bce5d35-7ced-46f5-8786-31eb932c1354
+US35,http://dbpedia.org/resource/Charles_Pomeroy_Stone,f06a78fa-f99c-4b7e-bea7-89aa4829c315
+US51,http://dbpedia.org/resource/Edward_Dickinson_Baker,4b479de3-ec5f-4531-af8e-903c593a7b78
+CS50,http://dbpedia.org/resource/J._E._B._Stuart,6df9b482-4757-474f-80a9-bd1d82228a63
+US57,http://dbpedia.org/resource/Edward_Ord,3f5fd4ed-d11f-4911-ae1a-5c241c74aa7c
+CS92,http://dbpedia.org/resource/Joseph_E._Johnston,b0f7a304-f6c5-4a8d-b816-066426e5dc1c
+US77,http://dbpedia.org/resource/George_B._McClellan,113829b2-e1ff-4458-8dce-9a4da30810dd
+CS69,http://dbpedia.org/resource/John_B._Magruder,256b790f-0cad-46d6-80aa-7e176890da83
+US77,http://dbpedia.org/resource/George_B._McClellan,113829b2-e1ff-4458-8dce-9a4da30810dd
+CS59,http://dbpedia.org/resource/James_Longstreet,0418b77d-5826-4c1d-9621-9d92c48d77b5
+CS41,http://dbpedia.org/resource/Gustavus_Woodson_Smith,7ecf1477-925a-4b2b-b174-a549a4038ecc
+US194,http://dbpedia.org/resource/William_B._Franklin,a6b7f4dd-efc1-43ae-ba35-a53ed768fd63
+US66,http://dbpedia.org/resource/Fitz_John_Porter,3e3bcb35-5488-46d2-8cf2-b0243ca67d45
+US77,http://dbpedia.org/resource/George_B._McClellan,113829b2-e1ff-4458-8dce-9a4da30810dd
+CS92,http://dbpedia.org/resource/Joseph_E._Johnston,b0f7a304-f6c5-4a8d-b816-066426e5dc1c
+CS41,http://dbpedia.org/resource/Gustavus_Woodson_Smith,7ecf1477-925a-4b2b-b174-a549a4038ecc
+US77,http://dbpedia.org/resource/George_B._McClellan,113829b2-e1ff-4458-8dce-9a4da30810dd
+CS122,http://dbpedia.org/resource/Robert_E._Lee,d428cfef-5c7a-4230-b394-b195481d9a9f
+CS122,http://dbpedia.org/resource/Robert_E._Lee,d428cfef-5c7a-4230-b394-b195481d9a9f
+US66,http://dbpedia.org/resource/Fitz_John_Porter,3e3bcb35-5488-46d2-8cf2-b0243ca67d45
+US66,http://dbpedia.org/resource/Fitz_John_Porter,3e3bcb35-5488-46d2-8cf2-b0243ca67d45
+CS122,http://dbpedia.org/resource/Robert_E._Lee,d428cfef-5c7a-4230-b394-b195481d9a9f
+US77,NA,NA
+CS69,NA,NA
+US59,http://dbpedia.org/resource/Edwin_Vose_Sumner,f2b24743-4d9d-4422-b830-26ecf22d9bd1
+CS69,http://dbpedia.org/resource/John_B._Magruder,256b790f-0cad-46d6-80aa-7e176890da83
+NA,NA,NA
```

Comment:
I'm not sure about the empty lines. I'll fix the other stuff and take a
look.

On Fri, Aug 5, 2016 at 9:57 PM, Jeffrey Arnold <notifications@github.com>
wrote:

> In rawdata/misc/commanders_to_ach_generals.csv
> <https://github.com/jrnold/acw_battle_data/pull/50#discussion_r73780955>:
>
> > +US194,http://dbpedia.org/resource/William_B._Franklin,a6b7f4dd-efc1-43ae-ba35-a53ed768fd63
> > +US66,http://dbpedia.org/resource/Fitz_John_Porter,3e3bcb35-5488-46d2-8cf2-b0243ca67d45
> > +US77,http://dbpedia.org/resource/George_B._McClellan,113829b2-e1ff-4458-8dce-9a4da30810dd
> > +CS92,http://dbpedia.org/resource/Joseph_E._Johnston,b0f7a304-f6c5-4a8d-b816-066426e5dc1c
> > +CS41,http://dbpedia.org/resource/Gustavus_Woodson_Smith,7ecf1477-925a-4b2b-b174-a549a4038ecc
> > +US77,http://dbpedia.org/resource/George_B._McClellan,113829b2-e1ff-4458-8dce-9a4da30810dd
> > +CS122,http://dbpedia.org/resource/Robert_E._Lee,d428cfef-5c7a-4230-b394-b195481d9a9f
> > +CS122,http://dbpedia.org/resource/Robert_E._Lee,d428cfef-5c7a-4230-b394-b195481d9a9f
> > +US66,http://dbpedia.org/resource/Fitz_John_Porter,3e3bcb35-5488-46d2-8cf2-b0243ca67d45
> > +US66,http://dbpedia.org/resource/Fitz_John_Porter,3e3bcb35-5488-46d2-8cf2-b0243ca67d45
> > +CS122,http://dbpedia.org/resource/Robert_E._Lee,d428cfef-5c7a-4230-b394-b195481d9a9f
> > +US77,NA,NA
> > +CS69,NA,NA
> > +US59,http://dbpedia.org/resource/Edwin_Vose_Sumner,f2b24743-4d9d-4422-b830-26ecf22d9bd1
> > +CS69,http://dbpedia.org/resource/John_B._Magruder,256b790f-0cad-46d6-80aa-7e176890da83
> > +NA,NA,NA
>
> @weszuidema <https://github.com/weszuidema> Delete empty lines. Do you
> know what is going on with this line?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/jrnold/acw_battle_data/pull/50/files/3b672ca30f2571eb94b9f1435ed18cba97eb5374#r73780955>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AP1WB8f7HvHkWB7utqBq7AyUMyM9t4Xmks5qdBRXgaJpZM4JeNBQ>
> .
>